tile-generation/src/test/scala/com/oculusinfo/tilegen/pipeline/PipelineOperationsTests.scala

```diff
@@ -675,4 +738,43 @@ class PipelineOperationsTests extends FunSuite with SharedSparkContext with Tile
 			removeRecursively(new File("test.x1.y1.x2.y2.data"))
 		}
 	}
+
+  test("Test Load Parquet File") {
+
+    def SaveParquetDataOp(path: String)(input: PipelineData): PipelineData = {
+      input.srdd.saveAsParquetFile(path)
+      PipelineData(input.sqlContext, input.srdd)
+    }
+
+    val resultList = ListBuffer[Any]()
+    val argMap = Map(
+      "ops.path" -> getClass.getResource("/json_test.data").toURI.getPath,
+      "ops.columns" -> "num,num_1",
+      "ops.min" -> "2,2",
+      "ops.max" -> "3,3")
+
+    val tempFolder = "./temp-" + System.currentTimeMillis()
```

Comment:
there is a File.createTempFile method that you can use to guarantee uniqueness, instead of just appending the system time.